Makefile

```diff
@@ -150,3 +147,16 @@ node_modules/.uptodate: package.json
 	@echo installing javascript dependencies
 	@node_modules/.bin/check-dependencies 2>/dev/null || npm install
 	@touch $@
+
+PYTHON_VERSION = 3.6.8
+PYTHON_MINOR_VERSION := $(shell echo $$PYTHON_VERSION | cut -d '.' -f1,2)
```

Comment:
So `PYTHON_MINOR_VERSION` is `3.6`. This is needed because the executable is just called `python3.6` (there is no `python3.6.8` executable) but the full `3.6.8` is needed for `pyenv install 3.6.8` (you can't just do `pyenv install 3.6`, and we want to specify the exact version anyway)